changelog.json

```diff
@@ -203,9 +204,7 @@
       "[Improved] Performance updating menu items - #1321",
       "[Improved] Windows: Dim the title bar when the app loses focus - #1189"
     ],
-    "0.0.39": [
-      "[Fixed] An uncaught exception when adding a user - #1394"
-    ],
+    "0.0.39": ["[Fixed] An uncaught exception when adding a user - #1394"],
```

Comment:
We seem to be doing some prettifying in there (at least in VSCode). Not sure if I should revert this.